tasks/main.yml

```diff
@@ -57,37 +57,62 @@
         dest: /etc/systemd/system/amazon-cloudwatch-agent.service.d/override.conf
         mode: 0644
         src: override.conf
-      notify: "systemd daemon-reload"
 
 # The AWS CloudWatch Agent systemd unit kicks off a process that
 # starts the CloudWatch Agent and then dies.  Therefore we can't start
 # it here because it will be started again during the idempotence test
 # and therefore will fail idempotence.
 - name: Enable AWS CloudWatch Agent
-  ansible.builtin.service:
+  ansible.builtin.systemd_service:
+    daemon_reload: true
```

Comment:
This change mean that this task will _always_ perform a daemon reload regardless of if a change will occur. Just making sure this is intended since that's technically a change from previous behavior.

Reply:
Please see commit b6766b4.

Reply:
Excellent, thanks!